spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
So if we have somebody asking about `use-package` here is how we should explain the lazy-loading:

- if `:defer` or `:commands` is used then the package is lazy loaded
- if not then it is required right away.

And we avoid the confusion with `:init` and `:config` which are identical when the package is not lazy-loaded.